Zend/zend_globals.h

```diff
@@ -81,7 +81,14 @@ struct _zend_compiler_globals {
 
 	HashTable *auto_globals;
 
-	zend_bool parse_error;
+	/* From zend_language_parser.y:
+	 * CG(parse_error) states:
+		* 0 => yyres = NULL, yystr is the unexpected token
+		* 1 => yyres = NULL, yystr is one of the expected tokens
+		* 2 => yyres != NULL, yystr is the unexpected token
+		* 3 => yyres != NULL, yystr is one of the expected tokens
+	 */
```

Comment:
`/* See zend_yytnamerr() for meaning of values */`

yyres / yystr don't make much sense out of context here.

Reply:
ACK